### ISSUE TITLE
In-body unsubscribe not recognized in email test

### ISSUE THREAD

**Issue by DivaJosie247** [wttw/aboutmyemail#15]
Hello,

I sent an email to wheel.lady.apricot@aboutmy.email from Unifund. Please see screenshots of the unsub link in the email and the test results attached.

Thanks very much for your help.

Make it a great day!

Josie Garcia
Deliverability & Operations Manager
TrueAccord
josie.garcia@trueaccord.com

![Unsubscription1](https://github.com/wttw/aboutmyemail/assets/155675749/cc2b2b55-e23a-4208-a416-d3232502dc7a)

**Comment by wttw** [comment]
Yup, our heuristics are based only on the URL and the text of the link. I'm not sure how best to handle this case (maybe look at the DOM object surrounding the link?).

**Comment by DivaJosie247** [comment]
Thanks so much, Steve! Send my love to Laura!



Make it a great day!


Garcia Out~



Josie Garcia

Deliverability & Operations Manager


<http://www.trueml.co/>

www.trueml.co

16011 College Blvd, Ste. 130  |  Lenexa, KS 66219

<https://www.linkedin.com/company/truemlco/about/>
<https://twitter.com/TrueMLco>
